### ISSUE TITLE
switchTO should shows the correct user message.

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#848]
### Expected behavior:

**switchTo:**
 `switchTo` api only takes either `url` or `page title`.
     If we use `switchTo` api without url or page title then it should show the user friend message to define the url.

### Actual behavior
currently it shows the below message:
```
> switchTo()
 ✔ Switched to tab with URL undefined
```

### Steps to reproduce
```
> openBrowser()
> goto("google.com")
> openTab("amazon.com")
> openTab()
> switchTo()

```

### Versions
```
Version: 1.0.2 (Chromium: 76.0.3803.0) 22a7d58c629864e04042d4661046d0c3e8e802cd
```

**Comment by BugDiver** [comment]
When there is no URL given it switches to the first opened tab. This was done intentionally.

**Comment by BugDiver** [comment]
Fix availabe in 4aa423fae783d8be07387120da7773606dc50db8

**Comment by Debashis9012** [comment]
Below is the current behaviour of `switchTo` api without url/page title:
`> switchTo()
 ✘ Error: The "targetUrl" argument must be of type string. Received type undefined, run `.trace` for more info.
`
based on the observation issue has been verified and found fixed.
Tested version:
`Version: 1.0.3 (Chromium: 76.0.3803.0) 4aa423fae783d8be07387120da7773606dc50db8`
